core/src/main/java/org/apache/gravitino/catalog/EntityCombinedModel.java

```diff
@@ -40,6 +40,14 @@ private EntityCombinedModel(Model model, ModelEntity modelEntity) {
     this.modelEntity = modelEntity;
   }
 
+  public ModelEntity getModelEntity() {
```

Comment:
The style is weird. We don't use getter. It's better to use `entity()` or `modelEntity()`.